### PR TITLE
[API] Add docs for scheduling a report

### PR DESCRIPTION
**Pull request by NickLaMuro**
Added these since they did not exist previously.

Links
-----

* Asked about in `gitter`: https://gitter.im/ManageIQ/api?at=5fc0d2446ebe7532e6a5f226
* Feature added to the API here:  https://github.com/ManageIQ/manageiq/pull/8647

### CODE REVIEW

**Comment by Fryguy** [comment]
Separately don't forget to add it to the website here: https://github.com/ManageIQ/manageiq.org/blob/master/site/_data/menus/api_menu.yml

Yes, I know I have to move it, and no, I haven't had a chance yet 😅

**Review by Fryguy — APPROVED** [review]
@abellotti Please also review.

**Comment by NickLaMuro** [comment]
> Separately don't forget to add it to the website here...

Uhh... doesn't it already exist?

> EDIT: Oh, for some reason I thought it was a brand new page and not updating the existing page...you can ignore 😄

oh good, I wasn't going crazy... 😅

**Review by kbrock — APPROVED** [review]
thanks for putting this together

File: api/reference/reports.md [hunk 1]
```diff
@@ -86,6 +88,47 @@ Example for running a report on the following:
...
+    "resource_type": "MiqReport",
+    "start_date": "2000-01-01T00:00:00Z",
+    "time_zone": "UTC",
+    "interval": { "unit": "once" },
+    "enabled": true
+  }
+}
+```
+
+
```
Comment (abellotti) [inline api/reference/reports.md:131]: this new section should go below the "Running Reports" section, probably right before "Requesting Report Result Downloads" around line 363.   Also, maybe add the Request label and show the related Response. Thanks.

**Comment by NickLaMuro** [comment]
@abellotti I think I completely missed this review a few months back, but yeah, I screwed up and tossed this in the middle of another section (a big reason why I prefer the `=====`/`-----` style `h1`/`h2` headers myself...).

I think I now have this where you want it, and also had to dig up how I was making the request to replicate myself a response, but that should be there as well.

Hopefully this covers all of your suggestions, but let me know if there is anything that needs to be tweaked.  Thanks.

**Comment by Fryguy** [comment]
Going to merge.  @abellotti if you see anything outstanding comment anyway, and it can be fixed in a followup

**Pull request merged**